core/templates/utility/string-utility.ts

```diff
@@ -16,11 +16,40 @@
  * @fileoverview Utility functions for strings.
  */
 
-export default function camelCaseFromHyphen(str: string): string {
+export function camelCaseFromHyphen(str: string): string {
   const newStr = str.replace(/[\])}[{(]/g, '');
   return newStr.replace(
     /-([a-z])/g,
     function(g) {
       return g[1].toUpperCase();
     });
 }
+
+export function checkEditDistance(
+  inputString: string,
+  matchString: string,
+  reqdEditDistance: number
```

Comment:
```suggestion
  reqEditDistance: number
```

Reply:
Adding to this: can you just put required instead of the short form req.

Reply:
Thanks! Will do!